File: Scripts/laptop/classes/perl/classes.pl
package SampleObject;

use strict;
use warnings;

sub new {
    my ($class, %args) = @_;
    return bless { %args }, $class;
}

sub sample_method {
    my ($self) = @_;
  #  print $self->{sample_data}; 
#    print $self->{more_data};	
}

sub executeTask {
	my ($i) = @_;
	my $obj = SampleObject->new( sample_data => 'Variable value is ',
                             more_data   => "1\n" );
	$obj->sample_method();
	return ($i + 1);
}
my $r = 1;

for( $a=0; $a < 2100000000; $a = $a + 1) {
	$r = executeTask($a + $r);
}

File: Scripts/laptop/classes/python/classes.py
class MyClass:

   def __init__(self,  variable):
      self.variable = variable
   
   def displayMyClass(self):
	self.variable = 1

def executeTask(i):
	obj = MyClass( 2000)
	obj.displayMyClass()
	return i + 1

r = 1

for i in xrange(2100000000):
	r = executeTask(i + r)
